README.md

```diff
@@ -2,7 +2,7 @@
 
 ### Purpose
 
-A [protobuf](https://developers.google.com/protocol-buffers/) compiler plugin 
+A [protobuf](https://developers.google.com/protocol-buffers/) compiler plugin
```

Comment:
Is this space removal thing something that IDE automatically does?

Reply:
Yes, it's part of the formatting executed whenever I save a file.